scripts/e2e-email/localeQuirks.js

```diff
@@ -12,31 +12,76 @@ var translationQuirks = {
   'content-language': [
   ],
 
-  'Verify your account': [
-    'en'
+  'Verify your Firefox Account': [
+    'ca',
+    'cy',
+    'en',
+    'es-AR',
+    'eu',
+    'ff',
+    'he',
+    'hu',
+    'id',
+    'ko',
+    'lt',
+    'nb-NO',
+    'pa',
+    'pl',
+    'sk',
+    'sq',
+    'sr',
+    'sr-LATN',
+    'sv',
+    'tr'
   ],
 
-  'Reset your password': [
-    'en'
+  'Reset your Firefox Account password': [
+    'ca',
+    'cy',
+    'en',
+    'es-AR',
+    'eu',
+    'ff',
+    'he',
+    'hu',
+    'id',
+    'ko',
+    'lt',
+    'nb-NO',
+    'pa',
+    'pl',
+    'sk',
+    'sq',
+    'sr',
+    'sr-LATN',
+    'sv',
+    'tr'
   ],
 
   // A *lot* have not been translated, but a number of broadly
   // spoken languages (e.g., de, zh-TW, zh-TW, ja, ru, pt-BR)
   // have been.
-  'Re-verify your account': [
+  'Re-verify your Firefox Account': [
     'ca',
+    'cy',
     'en',
     'es-AR',
+    'eu',
     'ff',
     'he',
+    'hu',
     'id',
     'ko',
     'lt',
     'nb-NO',
     'pa',
     'pl',
-    'rm',
+    'sk',
+    'sq',
+    'sr',
+    'sr-LATN',
     'sv',
+    'tr'
```

Comment:
Late to the party here, but it looks like this array of locales is repeated 3x. Should we abstract out to a `const` to keep them in sync, or would these 3 arrays always be slightly different based on "conditions"?